schema/attributegroup.cpp

```diff
@@ -71,4 +71,14 @@ Attribute::List AttributeGroup::attributes() const
 {
     return d->mAttributes;
 }
+
+bool operator==(const AttributeGroup &lhs, const AttributeGroup &rhs)
+{
+    return ( // XmlElement:
```

Comment:
That's the duplication I'm referring to. This part should just be done once, in `XmlElement::operator==`